bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/java/org/openhab/binding/modbus/helioseasycontrols/internal/HeliosEasyControlsActions.java

```diff
@@ -47,60 +47,80 @@ public void setThingHandler(@Nullable ThingHandler handler) {
 
     private void triggerSwitch(String variableName) {
         try {
-            if (handler != null) {
-                handler.writeValue(variableName, "1");
+            if (this.handler != null) {
+                this.handler.writeValue(variableName, "1");
             }
         } catch (HeliosException e) {
             logger.warn("Error executing action 'resetFilterChangeTimer': {}", e.getMessage());
         }
     }
 
-    @RuleAction(label = "reset filter change timer", description = "Sets the filter change timer back to the configured interval.")
+    @RuleAction(label = "Reset filter change timer", description = "Sets the filter change timer back to the configured interval")
     public void resetFilterChangeTimer() {
-        triggerSwitch(HeliosEasyControlsBindingConstants.FILTER_CHANGE_RESET);
+        this.triggerSwitch(HeliosEasyControlsBindingConstants.FILTER_CHANGE_RESET);
```

Comment:
Revert

Reply:
These changes were the result of  #8755 and should therefore remain.